composer.json

```diff
@@ -19,24 +19,32 @@
              "DigipolisGent\\CommandBuilder\\Test\\": "tests"
         }
     },
+    "require": {
+        "php": "^7.4 || ^8.0"
+    },
     "require-dev": {
-        "phpunit/phpunit": "^8",
-        "squizlabs/php_codesniffer": "^2.3",
-        "phpro/grumphp": "^0.15.2",
-        "sebastian/phpcpd": "^4.1",
-        "phpmd/phpmd": "^2.7",
-        "phpstan/phpstan": "^0.11.15",
-        "sensiolabs/security-checker": "^6.0"
+        "digipolisgent/qa-drupal": "^1.7"
```

Comment:
This is not a Drupal module or site.
Use the https://packagist.org/packages/district09/qa-php instead.